packages/ethers/src/client.ts

```diff
@@ -836,7 +839,10 @@ export class Web3Modal extends Web3ModalScaffold {
         EthersStoreUtil.setProvider(config.injected)
         EthersStoreUtil.setStatus('connected')
         EthersStoreUtil.setIsConnected(true)
-        this.setAllAccounts(addresses.map(address => ({ address, type: 'eoa' })))
+        this.setAllAccounts(
+          addresses.map(address => ({ address, type: 'eoa' })),
+          this.chain
```

Comment:
These are just to make sure we are passing the active chain to the scaffold, this will help us when we switch to multi-chain architecture.